rct229/rulesets/ashrae9012019/section5/section5rule25.py

```diff
@@ -110,5 +110,9 @@ def get_calc_vals(self, context, data=None):
                 }
 
             def rule_check(self, context, calc_vals=None, data=None):
+                skylight_roof_ratio_b = calc_vals["skylight_roof_ratio_b"]
+                return skylight_roof_ratio_b == 0.03
+
+            def is_tolerance_fail(self, context, calc_vals=None, data=None):
                 skylight_roof_ratio_b = calc_vals["skylight_roof_ratio_b"]
                 return std_equal(skylight_roof_ratio_b, 0.03)
```

Comment:
I prefer to put this 0.03 to a enum. The class should have an enum `SKYLIGHT_THRESHOLD`. Use that instead of 0.03.